src/schemas/chat.py

```
@@ -26,6 +26,18 @@ class ChatRequest(BaseModel):
     country_id: int  # 검색 대상 국가 ID (예: 1: California, 2: New York)
 
+    # 대화 세션 ID (대화 맥락 기억 기능용)
+    # - 프론트엔드에서 UUID를 생성하여 전달합니다.
+    # - 같은 session_id로 요청하면 이전 대화를 기억합니다.
+    # - None이면 맥락 없이 단독 질문으로 처리합니다. (기존 동작과 동일)
+    session_id: Optional[str] = None
+
+    @field_validator("query")
+    def check_query(cls, v):
+        if not v or not v.strip():
+            raise ValueError("질문을 입력해주세요.")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)